modules/nf-core/dysgu/main.nf

```diff
@@ -0,0 +1,55 @@
+process DYSGU {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/dysgu:48830f55112c399e':
+        'community.wave.seqera.io/library/dysgu:faf71ac972284412' }"
+
+    input:
+    tuple val(meta), path(input_bam), path(input_bam_index)
```

Comment:
The tool takes cram files too, given your test, so maybe call this something more generic? Can't recall off the top of my head what we typically use to mean bam/cram.

Reply:
checked main.nf of manta instead of `input_bam` here it's described just `input` 
 will this work? 
 ```
input:
   
    tuple val(meta), path(input), path(index), path(target_bed), path(target_bed_tbi)
    tuple val(meta2), path(fasta)
    tuple val(meta3), path(fai)
    path(config)
```

Reply:
`input` and `index` sounds fine.

Reply:
change it and update in the meta accordingly